### PR TITLE
Enable Indexing of Courseware and Libraries

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @kluo

### CODE REVIEW

**Comment by kluo** [comment]
+1

File: cms/envs/common.py [hunk 1]
```diff
@@ -160,10 +160,10 @@
     'LICENSING': False,
 
     # Enable the courseware search functionality
-    'ENABLE_COURSEWARE_INDEX': False,
+    'ENABLE_COURSEWARE_INDEX': True,
```
Comment (stvstnfrd) [inline cms/envs/common.py]: What are the defaults in `configuration`?

Would those override this?

Comment (caesar2164) [inline cms/envs/common.py]: These settings are NOT defaulted anywhere in configuration.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**